src/styles/blocks/card.scss

```diff
@@ -0,0 +1,159 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the gap property should have a space before the second value: `gap: 46px 48px;` should be `gap: 46px  48px;`.